frontend/src/components/app/app.tsx

```diff
@@ -90,7 +90,7 @@ const App: React.FC = () => {
           component={EpisodeLive}
           exact
         />
-        <AuthPublicRouter path={AppRoute.ROOT} component={Homepage} exact />
+        <AuthPublicRouter path={[AppRoute.ROOT, AppRoute.ROOT_$QUERY]} component={Homepage} exact />
```

Comment:
```suggestion
        <AuthPublicRouter path={AppRoute.ROOT} component={Homepage} exact />
```
params should be like this - https://jabber-my.space/?title=moreover